docs/docs/references/configuration/cli/trivy_image.md

```diff
@@ -15,6 +15,11 @@ trivy image [flags] IMAGE_NAME
   # Scan a container image from a tar archive
   $ trivy image --input ruby-3.1.tar
 
+  # Scan an OCI imag from a local layout referenced by tag or digest
```

Comment:
This documentation is auto-generated by `mage docs:generate`. Instead, can you please update the following document?
https://github.com/aquasecurity/trivy/blob/7e99d08a131c05adf8d6f70a878c5a943a8f2d7a/docs/docs/target/container_image.md#oci-layout

Reply:
Alright, I have moved the examples to the document you linked.